aim/sdk/objects/image.py

```diff
@@ -25,9 +25,9 @@ class Image(CustomObject):
     Args:
          image (:obj:): pillow `Image` object or `torch.Tensor` or `numpy.array` used to construct `aim.Image`.
          caption (:obj:`str`, optional): Optional image caption. '' by default.
-         format (:obj: `str`, optional): Parameter for PIL's .save() method. 'png' by default.
-         quality (:obj: `int`, optional): Parameter for PIL's .save() method. 85 by default.
-         optimize (:obj: `bool`, optional): Parameter for PIL's .save() method. False by default.
+         format (:obj:`str`, optional): Parameter for PIL's .save() method. 'png' by default.
+         quality (:obj:`int`, optional): Parameter for PIL's .save() method. 90 by default.
```

Comment:
Otherwise, I'll replace the default value `90` with `85`.

Reply:
`90` is the correct value. Seems the docstring wasn't updated after the change.

Reply:
Thank you for your check!